Disable scuba logging for autotuning

masnesral opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #144568

Summary: the compile IDs are currently null, which is confusing. Turn it off until we have a solution.

Test Plan: https://fburl.com/scuba/dynamo_compile/sandbox/g2d2g5xs

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang @aakhundov

masnesral committed:
Disable scuba logging for autotuning

Summary: the compile IDs are currently null, which is confusing. Turn it off until we have a solution.

Test Plan: https://fburl.com/scuba/dynamo_compile/sandbox/g2d2g5xs

[ghstack-poisoned]

jamesjwu commented on torch/_inductor/runtime/triton_heuristics.py:
```diff
@@ -761,7 +761,8 @@ def benchmark_all_configs(self, *args, **kwargs):
             metadata={"kernel_name": self.inductor_meta.get("kernel_name")},
-            dynamo_compile_runtime_column_us="runtime_triton_autotune_time_us",
+            # TODO(masnesral): Enable this when we figure out how to get the CompileId:
+            # dynamo_compile_runtime_column_us="runtime_triton_autotune_time_us",
```
In the end, we only really care about disabling it for tlparse due to the unnecessary spew, so you could have also just turned it off on that side and kept the dynamo_compile rows. But this is fine

masnesral commented:
@pytorchbot merge

masnesral committed:
Update on "Disable scuba logging for autotuning"


Summary: the compile IDs are currently null, which is confusing. Turn it off until we have a solution.

Test Plan: https://fburl.com/scuba/dynamo_compile/sandbox/g2d2g5xs

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire chauhang aakhundov

[ghstack-poisoned]

masnesral commented:
@pytorchbot merge
